plugins/inputs/nvidia_smi/README.md

```diff
@@ -0,0 +1,155 @@
+# Nvidia System Management Interface (SMI) Input Plugin
+
+This plugin uses a query on the
+[`nvidia-smi`](https://developer.nvidia.com/nvidia-system-management-interface)
+binary to pull GPU stats including memory and GPU usage, temp and other.
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
+
+## Configuration
+
+```toml @sample.conf
+# Pulls statistics from nvidia GPUs attached to the host
+[[inputs.nvidia_smi]]
+  ## Optional: path to nvidia-smi binary, defaults "/usr/bin/nvidia-smi"
+  ## We will first try to locate the nvidia-smi binary with the explicitly specified value (or default value),
+  ## if it is not found, we will try to locate it on PATH(exec.LookPath), if it is still not found, an error will be returned
+  # bin_path = "/usr/bin/nvidia-smi"
+
+  ## Optional: specifies plugin behavior regarding missing nvidia-smi binary
+  ## Available choices:
+  ##   - error: telegraf will return an error on startup
+  ##   - ignore: telegraf will ignore this plugin
+  # startup_error_behavior = "error"
+
+  ## Optional: timeout for GPU polling
+  # timeout = "5s"
+```
+
+### Linux
+
+On Linux, `nvidia-smi` is generally located at `/usr/bin/nvidia-smi`
+
+### Windows
```

Comment:
Same as the other comment, probably should remove windows related comments.

Reply:
I prefer to minimize differences with upstream. Also consider customers do not enable this plugin directly, they have to configure CWAgent using the JSON config. I think the JSON config translate into TOML and YAML and will ignore this input plugin for Windows.

Reply:
I recommend keeping the README.md file the same as upstream.